guides/common/modules/con_introduction-to-salt.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Introduction_to_Salt_{context}"]
+= Introduction to Salt
+
+You can use Salt as a configuration management tool similar to {ManagingConfigurationsAnsibleDocURL}[Ansible] or {ManagingConfigurationsPuppetDocURL}[Puppet].
+
+This guide describes how to use Salt for configuration management in {Project}.
+This guide contains information about how to install the Salt plug-in, how to integrate {Project} with an existing Salt Master, and how to configure managed hosts with Salt.
+
+[NOTE]
+====
+Salt offers two distinct modes of operation:
+Clientless using SSH or the Salt Minion client software.
+
+ifdef::orcharhino[]
+{Project}'s Salt plug-in supports exclusively the Salt Minion approach.
+endif::[]
+====
+
+.Additional Resources
```

Comment:
Is this the place we were talking about on Monday? cc @asteflova

Reply:
Yes, that's it.